packages/extension/src/config/contracts/production.js

```diff
@@ -1,4 +1,4 @@
-import AccountRegistry from '~contracts/Behaviour20200106.json';
+import AccountRegistry from '~contracts/Behaviour20200207.json';
```

Comment:
Should this one and one above reference an interface instead, so that we can have a non-changing minimal API, and only need to update the SDK if and when the API actually changes?

Reply:
Makes sense. I'll check that the interface contract is up to date. 

What'd you think @joeandrews ?